### PR TITLE
Update dockerfile tags

### PR DESCRIPTION
**Pull request by bam241**
Fix my too-quick-merge on #765 

Tested [here]https://github.com/bam241/DAGMC/actions/runs/1057280538) (link updated for commit 7b8a1b0)

TEST badge ...
[Build & Publish docker image for DAGMC-CI](https://github.com/bam241/DAGMC/actions/workflows/docker_publish.yml)

### CODE REVIEW

File: CI/Dockerfile_1_external_deps [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ARG UBUNTU_VERSION=18.04ARG
 ARG OWNER=svalinn
-FROM ghcr.io/${OWNER}/dagmc-ci-ubuntu-${UBUNTU_VERSION}
+FROM ghcr.io/${OWNER}/dagmc-ci-ubuntu-${UBUNTU_VERSION}:ci_testing
```
Comment (gonuke) [inline CI/Dockerfile_1_external_deps]: Should we pass this in as an argument so that these scripts can be used in other contexts?

**Review by gonuke — COMMENTED** [review]
A question?

**Comment by bam241** [comment]
[Build & Publish docker image for DAGMC-CI](https://github.com/bam241/DAGMC/actions/workflows/docker_publish.yml)

**Comment by gonuke** [comment]
Thanks @bam241

**Pull request merged**